packages/components/src/Text/Text.tsx

```diff
@@ -93,7 +75,7 @@ interface BaseTypographyProps {
   size: TypographySize;
 }
 
-const TypographyComponent = styled.span<BaseTypographyProps>(
+export const TypographyComponent = styled.p<BaseTypographyProps>(
   ({ bold, color, size }) => [
     tw`font-arimo`,
```

Comment:
Are we going to specify this on every component? Should have folks specify this on the `<body>` tag, instead?

Reply:
yeah I thought about that, will do it in a follow-up